doc/releases/changelog-dev.md

```diff
@@ -4,92 +4,127 @@
+
+  * You can now easily access Pauli operators via `I`, `X`, `Y`, and `Z`:
+    [(#5116)](https://github.com/PennyLaneAI/pennylane/pull/5116)
+
+    ```pycon
+    >>> from pennylane import I, X, Y, Z
+    >>> X(0)
+    X(0)
+    ```
+
+  * PennyLane will try to automatically work with a Pauli representation of operators when
+    available. The Pauli representation can be optionally accessed via `op.pauli_rep`:
+    [(#4989)](https://github.com/PennyLaneAI/pennylane/pull/4989)
+    [(#5001)](https://github.com/PennyLaneAI/pennylane/pull/5001)
+    [(#5003)](https://github.com/PennyLaneAI/pennylane/pull/5003)
+    [(#5017)](https://github.com/PennyLaneAI/pennylane/pull/5017)
+    [(#5027)](https://github.com/PennyLaneAI/pennylane/pull/5027)
+  
+    ```pycon
+    >>> op = X(0) + Y(0)
+    >>> type(op.pauli_rep)
+    pennylane.pauli.pauli_arithmetic.PauliSentence
+    ```
+    
+    The `PauliWord` and `PauliSentence` objects in the
+    [pauli](https://docs.pennylane.ai/en/stable/code/qml_pauli.html#classes) module provide an
+    efficient representation and can be combined using basic arithmetic like addition, products, and
+    scalar multiplication. These objects do not need to be directly handled in most workflows
+    since manipulation will happen automatically in the background.
+
+  * Extensive improvements have been made to the string representations of PennyLane operators,
+    making them shorter and possible to copy as valid PennyLane code:
+    [(#5116)](https://github.com/PennyLaneAI/pennylane/pull/5116)
+    [(#5138)](https://github.com/PennyLaneAI/pennylane/pull/5138)
+
+    ```
+    >>> 0.5 * X(0)
+    0.5 * X(0)
+    >>> 0.5 * (X(0) + Y(1))
+    0.5 * (X(0) + Y(1))
+    ```
+    
+    Sums with many terms are broken up into multiple lines, but can still be copied back as valid
+    code:
+
+    ```
+    >>> 0.5 * (X(0) @ X(1)) + 0.7 * (X(1) @ X(2)) + 0.8 * (X(2) @ X(3))
+    (
+        0.5 * (X(0) @ X(1))
+      + 0.7 * (X(1) @ X(2))
+      + 0.8 * (X(2) @ X(3))
+    )
+    ```
+  
+  * The `Sum` and `Prod` classes have been updated to reach feature parity with `Hamiltonian`
+    and `Tensor`, respectively. This includes support for grouping via the `pauli` module:
+    [(#5070)](https://github.com/PennyLaneAI/pennylane/pull/5070)
+    [(#5132)](https://github.com/PennyLaneAI/pennylane/pull/5132)
+    [(#5133)](https://github.com/PennyLaneAI/pennylane/pull/5133)
+    
+    ```pycon
+    >>> obs = [X(0) @ Y(1), Z(0), Y(0) @ Z(1), Y(1)]
+    >>> qml.pauli.group_observables(obs)
+    [[Y(0) @ Z(1)], [X(0) @ Y(1), Y(1)], [Z(0)]]
+    ```
+
+  * A new `qml.commutator` function is now available that allows you to compute commutators between
+    operators:
+    [(#5051)](https://github.com/PennyLaneAI/pennylane/pull/5051)
+    [(#5052)](https://github.com/PennyLaneAI/pennylane/pull/5052)
+
+    ```pycon
+    >>> qml.commutator(X(0), Y(0))
+    2j * Z(0)
+    ```
+
+<h4>New Clifford device 🦾</h4>
```

Comment:
Was the Drawing and specs work with transforms supposed to be above the clifford device?

Reply:
Drawing and specs is getting bumped to 0.36, but nicely we have the work with circuit importing to replace it.